examples/webgl_loader_gltf.html

```diff
@@ -40,17 +40,14 @@
 				scene = new THREE.Scene();
 
 				new RGBELoader()
-					.setDataType( THREE.UnsignedByteType )
+					.setDataType( THREE.FloatType )
```

Comment:
Should we consider to make `THREE.FloatType` the default similar to `EXRLoader`? I guess most users do not want the nearest filtered background.

Reply:
Short-answer: yes.

`UnsignedByteType` was made the default so as to support mobile devices under WebGL1.

It is now several years later, and it appears that linear-float filtering is well-supported in WebGL1, and by default in WebGL2.

I was initially concerned about the change to `THREE.FloatType` in 12 examples in this PR. But if the support is now widely-available, then that is OK, too.

Your call.

Reply:
Let's make a separate PR for this change.